pyperformance/data-files/benchmarks/bm_sphinx/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx==6.2.1
```

Comment:
Any reason to use 6.2? The CPython docs (currently) use 8.0.2, which is the version in which the regression was noticed.

Reply:
It doesn't require installing a separate, newer version of docutils just for this benchmark (saves some time).  I picked it because it's the current minimum required version for CPython, but that's kind of arbitrary.

Reply:
I might suggest `Sphinx==7.3.7` then, it supports Docutils 18.1 (used in `bm_docutils`) and also the longer and less cryptic command line options I suggested above.

However as you say the choice is somewhat arbitrary and more important to have the benchmark.

A

Reply:
Thanks.  That makes sense.